### ISSUE TITLE
AVR-153 Dicecloud import does not work with custom spells.

### ISSUE THREAD

**Issue by InflamedSebi** [avrae/avrae#175]
Reproduce:
Import a character from dicecloud
a) 
1. Add a spell in dicecloud i.e. "Mending" (cantrip)
2. Rename the spell to something else like "Minor Repair"
3. "!update"
4. "!spellbook" does neither list "Mending" nor "Minor Repair" and you cannot cast it.
B) 
1. Add a spell in dicecloud i.e. "Mending" (cantrip)
2. Change properties of the spell like level/description/damage
3. "!update"
4. "!cast Mending" will cast the original mending spell at level cantrip and will use all original properties of the spell, instead of using the adjusted ones.

Does this mean dicecloud spell system is not supported and only names are getting imported?
How to add custom spells?

How is is supposed to work:
1. "!spellbook" should show all spells you have listed, with their adjusted name.
2. "!cast X" should cast the spell like stated in the spells tab, so use the level/description from there.

**Comment by athematici** [comment]
**TL;DR - it seems to be Dicecloud's fault**

I know this is nearly three years old at this point but it seems no sophistication has been added to this. Based on my current experience trying to set up a Mystic on Dicecloud, it appears that any spells, whether custom or SRD; spell attack or save, will be listed as attacks if you set up a macro for the spell in Dicecloud. Rather than Avrae, it seems the flaw belongs to Dicecloud because it only gives the options "Add Attack" and "Add Buff". Thus, when Avrae reads the spells, they will be listed under `!spellbook` but to actually cast them you would have to use `!attack` where you would normally use `!cast`.

**Comment by mommothazaz123** [comment]
Holy necro, Batman 

(Yes, if something is set up as an attack in Dicecloud, it'll be imported as an attack in Avrae, regardless of source. Whether or not something is a "spell" is if the name matches a known spell in Avrae's database)
